test/parallel/test-util-colorText.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const util = require('util');
+
+[
+  undefined,
+  null,
+  false,
+  5n,
+  5,
+  Symbol(),
+].forEach((invalidOption) => {
+  assert.throws(() => {
+    util.colorText(invalidOption, 'test');
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE'
+  });
+  assert.throws(() => {
+    util.colorText('red', invalidOption);
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE'
+  });
+});
+
+assert.throws(() => {
+  util.colorText('red', undefined);
+}, {
+  code: 'ERR_INVALID_ARG_TYPE',
+  message: 'The "text" argument must be of type string. Received undefined'
+});
```

Comment:
Wouldn't this already be tested on line 20 when `invalidOption` is `undefined`?

Reply:
Right, this should rather be `util.colorText(undefined, undefined);`